tests/codeception/acceptance/Content/BasicPageCest.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+use Faker\Factory;
```

Comment:
Oh, this is neat.

Reply:
yeah the faker factory does alot of nice random things that are nicely formatted. like names, addresses, links etc. its really nice.